package.json

```diff
@@ -356,7 +356,7 @@
     "moveable": "0.53.0",
     "nanoid": "^5.0.4",
     "node-forge": "^1.3.1",
-    "ol": "7.4.0",
+    "ol": "10.3.1",
```

Comment:
wow, from 7 to 10 major version 🤯

Reply:
yeah...

- here's the v8 upgrade PR: https://github.com/grafana/grafana/pull/82868
- here's the issue to track upgrading to v8: https://github.com/grafana/grafana/issues/82947
- here's v9: https://github.com/grafana/grafana/pull/86748
- now here we are almost a year later on v10 😅